platform_versioned_docs/version-24.1/launch/advanced.mdx

```diff
@@ -27,6 +22,58 @@ For example, modify the **manifest** section to give the pipeline a name and des
     }
 ```
 
+#### Nextflow configuration order of priority
+
+When launcing pipelines in Platform, Nextflow configuration values can be supplied from the `nextflow.config` file in the pipeline repository and the **Nextflow config file** field in the pipeline launch form. If different values of the same configuration parameter are defined, Nextflow parameters defined in the launch form **Nextflow config file** field override the same parameters in your `nextflow.config` file. 
+
+Configuration values set in the **Global Nextflow config** field during compute environment creation are pre-filled in the **Nextflow config file** field during pipeline launch. These pre-filled values from the compute environment can be overridden manually during launch. 
+
+For example, if:
+
+1. The `nextflow.config` file in your pipeline repository contains this manifest: 
+
+```ini
+    manifest {
+        name = 'A'
+        description = 'Description A'
```

Comment:
I wonder if something like

> pipeline repository example description

or something more concrete might be helpful for this?